src/hipscat_import/command_line_arguments.py

```diff
@@ -123,6 +123,17 @@ def parse_command_line(cl_args):
         dest="overwrite",
         action="store_false",
     )
+    group.add_argument(
+        "--resume",
+        help="if set, the pipeline will try to resume from a previous failed pipeline progress",
+        action="store_true",
+    )
+    group.add_argument(
+        "--no_resume",
+        help="if set, the pipeline will exit if existing intermediate files are found",
+        dest="resume",
+        action="store_false",
+    )
```

Comment:
Is it worth having an explicit "clean_resume" flag? If there are intermediate files the job will either continue or exit (which makes sense). In the case it exits, I think the user needs to clean the files manually. Adding a path to do that automatically could save a step in the future.